src/app/workspace/resource/representation/text/text.component.html

```diff
@@ -2,15 +2,37 @@
     <!-- in case of an error -->
     <app-status [status]="404" [url]="src.fileValue.fileUrl" [representation]="'text'" *ngIf="failedToLoad"></app-status>
 
-    <button class="download" mat-button (click)="downloadText(src.fileValue.fileUrl)" [disabled]="failedToLoad">
-        <mat-icon>
-            file_download
-        </mat-icon>
-        Click to download
-    </button>
-    <button mat-button matTooltip="Replace text file" (click)="openReplaceFileDialog()">
-        <mat-icon>cloud_upload</mat-icon>
-    </button>
+    <div class="file-representation">
+        <div class="container">
+            <div class="contents">
+                <div class="icon">
+                    <mat-icon>
+                        description
```

Comment:
I've changed the icon for text files to "text_snippet" in #790 which should be ready for review soon. Can you change it to this preemptively please?

Reply:
Done.